vir-generate/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "svirpti-vir-generate"
```

Comment:
This should correspond to the name of the folder

Reply:
I agree.